.github/labeler.yml

```diff
@@ -102,3 +102,6 @@ rbd:
   - src/tools/rbd/**
   - src/tools/rbd_mirror/**
   - src/tools/rbd_nbd/**
+ceph-volume:
```

Comment:
Minor thing: just for consistency (although it's valid YAML), let's keep a blank line before the label.